apps/projects/app/script.js

```diff
@@ -282,6 +306,17 @@ async function syncTokens(state, { token }) {
  *                     *
  ***********************/
 
+const resetAppState = (nextState) => Object.assign({}, nextState)
```

Comment:
Not a spread syntax fan?

Reply:
Ah! Yes I am a big fan, and will change :)